Task 5.2/Task5.2/Entities/Catalog.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System.Text.RegularExpressions;
+
+namespace Task5._2.Entities
+{
+    public class Catalog
+    {
+        private Dictionary<string, Book> _books;
+
+        public Catalog()
+        {
+            _books = new Dictionary<string, Book>();
+        }
+
+        public void AddBook(string isbn, Book book)
+        {
+            if (book == null)
+            {
+                throw new ArgumentNullException(nameof(book));
+            }
+
```

Comment:
Not proper logics,  acceptable format is  XXX-X-XX-XXXXXX-X, but not ----XXXXXXXXXXXXX, but you don't take this into considereation

Reply:
Fixed